python/processor.py

```
@@ -352,10 +354,122 @@ def process_command(action: str, args: Dict[str, str]) -> Dict[str, Any]:
+        error_msg = {
+            'message': str(e),
+            'details': {
+                'input_file': args.get('input_path', 'N/A'),
+                'output_directory': args.get('output_dir', 'N/A'),
+                'error_type': e.__class__.__name__
+            }
+        }
+        return format_json_response('error', error=error_msg)
+
+
+def process_command(action: str, args: Dict[str, str]) -> Dict[str, Any]:
```

Ruff (0.14.3)

Redefinition of unused `process_command` from line 287

(F811)